### PR TITLE
feature: landing page and alert component

### PR DESCRIPTION
**Pull request by sunilvr**

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Looks good. Add some space between alert icon & message.

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.primary-btn-outlined{
+    background-color: white;
+    color: var(--primary-color);
+    border: 0.2rem solid var(--primary-color);
+}
+
+.secondary-btn-outlined{
+    background-color: white;
```
Comment (abhisheksahani-nova) [inline components/buttons/buttons.css]: Use variable everywhere for colors white and gray as well

Comment (sunilvr) [reply components/buttons/buttons.css]: Hi Abhishek
thanks for the review. Changes suggested by you and Aniket have been incorporated now

Comment (sunilvr) [reply components/buttons/buttons.css]: @abhisheksahani-nova Could you please approve the PR as well?

Comment (abhisheksahani-nova) [reply components/buttons/buttons.css]: Yess go ahead

**Review by abhisheksahani-nova — APPROVED** [review]

**Pull request merged**